### PR TITLE
Implement blog page

### PR DESCRIPTION
**Pull request by Sanjeewa001**

### CODE REVIEW

File: app/blog/page.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+"use client";
+import { Box, Container, Grid,  } from "@mui/material";
+import Banner from "@/components/blogPage/Banner1"
+import BlogCard from "@/components/blogPage/blogcard";
+import { blogData, bannerData } from '@/data/blogPage';
+import Image from "next/image";
+
```
Comment (iamarasekera) [inline app/blog/page.tsx:7]: Document the code

File: components/blogPage/Banner1.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { Typography,Grid,TextField,Button } from "@mui/material";
```
Comment (iamarasekera) [inline components/blogPage/Banner1.tsx:1]: Rename the file to something like MainBanner etc. Not Banner 1, 2, 3 etc.

File: data/blogPage.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
+export const blogData = [
+    //  asign data into blog card in blog page
+    {
+        id: 1,
+        image: "/images/blogPage/anuradhapura.webp",
+        user: "saman kumara",
```
Comment (iamarasekera) [inline data/blogPage.ts:6]: This should be author not user

Comment (Sanjeewa001) [reply data/blogPage.ts]: done

**Pull request merged**